core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -430,12 +433,33 @@
       "When a tablet server's SimpleTimer thread triggers to check idle"
           + " sessions, this configurable option will be used to evaluate update"
           + " sessions to determine if they can be closed due to inactivity"),
+  @Deprecated
   TSERV_READ_AHEAD_MAXCONCURRENT("tserver.readahead.concurrent.max", "16", PropertyType.COUNT,
-      "The maximum number of concurrent read ahead that will execute. This effectively"
-          + " limits the number of long running scans that can run concurrently per tserver."),
+      "This property is deprecated, use tserver.scan.executors.default.threads instead."),
+  @Deprecated
   TSERV_METADATA_READ_AHEAD_MAXCONCURRENT("tserver.metadata.readahead.concurrent.max", "8",
       PropertyType.COUNT,
-      "The maximum number of concurrent metadata read ahead that will execute."),
+      "This property is deprecated, use tserver.scan.executors.meta.threads instead."),
+  TSERV_SCAN_EXECUTORS_PREFIX("tserver.scan.executors.", null, PropertyType.PREFIX,
```

Comment:
I thought there was a problem with having a prefix that also had well-defined properties "beneath" it. Might only apply to table properties, though.